tests/common/TestRuntime.cs

```diff
@@ -177,13 +177,22 @@ public static bool CheckExactXcodeVersion (int major, int minor, int beta = 0)
 			watchOS = new { Major = 7, Minor = 1, Build = "18R5561" },
 		};
 
+		var twelvedot2b3 = new {
+			Xcode = new { Major = 12, Minor = 2, Beta = 3 },
+			iOS = new { Major = 14, Minor = 2, Build = "18B5072" },
+			tvOS = new { Major = 14, Minor = 2, Build = "18K5047" },
+			macOS = new { Major = 11, Minor = 0, Build = "?" },
```

Comment:
These tuples (records?) seem to point to the version info. Is it valid to have `Build = "?"`?

Reply:
yep when '?' is used we throw an exception IIRC

Reply:
Yes, "?" basically means something like "don't know, don't care, fix me if that changes"